nodesigner.go

```diff
@@ -44,6 +44,27 @@ func (n *nodeSigner) SignMessage(pubKey *btcec.PublicKey,
 	return sign, nil
 }
 
+// SignCompact signs a double-sha256 digest of the msg parameter under the
+// resident node's private key. The returned signature is a pubkey-recoverable
+// signature.
+func (n *nodeSigner) SignCompact(msg []byte) ([]byte, error) {
+
+	// Otherwise, we'll sign the dsha256 of the target message.
+	digest := chainhash.DoubleHashB(msg)
+
+	// Should the signature reference a compressed public key or not.
+	isCompressedKey := false
```

Comment:
Public keys within the protocol are _always_ meant to be handled or transferred in a compressed format.